### PR TITLE
Some bug fixes and very minor refactoring

### PR DESCRIPTION
**Pull request by T00fy**

### CODE REVIEW

File: src/world/WorldServer.cpp [hunk 1]
```diff
@@ -144,6 +144,11 @@ std::string readFileToString( const std::string& filename )
   // Close the file
   file.close();
 
+  // Remove all newlines from the file contents
+  fileContents.erase(std::remove(fileContents.begin(), fileContents.end(), '\n'), fileContents.end());
```
Comment (hkAlice) [inline src/world/WorldServer.cpp:148]: we use whitespaces within parentheses
-> `fileContents.erase( std::remove( fileContents.begin(), fileContents.end(), '\n' ), fileContents.end() );`

File: src/world/WorldServer.cpp [hunk 2]
```diff
@@ -144,6 +144,11 @@ std::string readFileToString( const std::string& filename )
...
+  fileContents.erase(std::remove(fileContents.begin(), fileContents.end(), '\r'), fileContents.end());
```
Comment (hkAlice) [inline src/world/WorldServer.cpp:149]: likewise here

**Review by hkAlice — COMMENTED** [review]
thank you for the PR, looks good - just some comments on coding style

File: src/world/WorldServer.cpp (hunk 2 above)
Comment (T00fy) [reply src/world/WorldServer.cpp]: done

**Review by hkAlice — APPROVED** [review]

**Pull request merged**